src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,68 @@
+import cn from 'classnames';
+import { FilterStatus } from '../../types/Filterstatus';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  completedTodos: Todo[] | [],
+  uncompletedTodosAmount: number,
+  onStatusChange: (str: FilterStatus) => void,
+  filterStatus: FilterStatus,
+};
+
+export const Footer: React.FC<Props> = ({
+  completedTodos,
+  uncompletedTodosAmount,
+  onStatusChange,
+  filterStatus,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${uncompletedTodosAmount} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className={cn('filter__link',
+            { selected: filterStatus === FilterStatus.All })}
+          onClick={() => onStatusChange(FilterStatus.All)}
```

Comment:
Fix here and in all other places
```suggestion
          onClick={() => setFilterStatus(FilterStatus.All)}
```